src/main/java/Leas_Liudmila/encapsulation/EmployeeMain.java

```diff
@@ -0,0 +1,40 @@
+package Leas_Liudmila.encapsulation;
+
+import java.util.ArrayList;
+
+public class EmployeeMain {
+    public static void main(String[] args) {
+        Employee manager = new Employee();
+        manager.setId(1);
+        manager.setName("Alex");
+        manager.setSalary(1255.6f);
+
+        Employee director = new Employee();
+        director.setId(2);
```

Comment:
in my opinion ID should be an final random number, not a thisng which you can set or reset